src/main.rs

```diff
@@ -20,32 +20,13 @@ fn main() -> Result<(), Box<dyn Error>> {
         process::exit(0);
     }
 
-    let is_fix = args.is_present("fix");
+    warnings.iter().for_each(|w| println!("{}", w));
```

Comment:
I think we shouldn't print warnings for `quiet` mode (or maybe even by default?), because we fix all the `Warning`s.

This seems to be the standard linter behavior, and we also have a flag to configure additiinal output :thinking:
What do you think?

Reply:
I'd like do not print them for quiet mode.

Reply:
How do you like these options?

Not in quiet mode:
```bash
$ dotenv-linter -f
Original file was backed up to: ".env_1601369083"

.env:3 LowercaseKey: The a0 key should be in uppercase
.env:4 LowercaseKey: The a2 key should be in uppercase

All warnings are fixed. Total: 2
```

In quiet mode:
```bash
$ dotenv-linter -f --quiet
Original file was backed up to: ".env_1601369083"

All warnings are fixed. Total: 2
```

Reply:
> I think we shouldn't print warnings for `quiet` mode (or maybe even by default?), because we fix all the `Warning`s.

I think the user should know what warnings were found and fixed without running `dotenv-linter` in check mode.

Reply:
"Not in quiet mode" is looking good.

But maybe don't print anything in quiet mode?

Reply:
> But maybe don't print anything in quiet mode?

The minimal information always should be in quiet mode.

Reply:
Then "In quiet mode" is OK too.

Reply:
> How do you like these options?


I have a little doubt about the output of backup files (in the case when there are many files, this is not so "`quiet`")
But yes, the user should know about the created backups (maybe they should just be placed in a special folder?)

So, I like the proposed options, thank you!

Reply:
> I have a little doubt about the output of backup files (in the case when there are many files, this is not so "`quiet`")
> But yes, the user should know about the created backups (maybe they should just be placed in a special folder?)

If you have an idea how we can improve the output of backup files, please open an issue and we will discuss it.